packages/@o3r/extractors/builders/aggregate-migration-scripts/index.ts

```diff
@@ -0,0 +1,169 @@
+import { BuilderOutput, createBuilder } from '@angular-devkit/architect';
+import { createBuilderWithMetricsIfInstalled, type MigrationFile } from '@o3r/extractors';
+import * as fs from 'node:fs';
+import { mkdir, readFile, rm, writeFile } from 'node:fs/promises';
+import { basename, dirname, join, posix, resolve } from 'node:path';
+import * as globby from 'globby';
+import * as semver from 'semver';
+import { AggregateMigrationScriptsSchema } from './schema';
+
+/** Maximum number of steps */
+const STEP_NUMBER = 3;
```

Comment:
Why 3? Is this a functional or a technical constraint?

Reply:
Okay, just got to the part where it is used. Instead of having a step number, why don't we just list the steps and use the step length instead ?